test/hotspot/jtreg/compiler/vectorapi/VectorReplicateLongSpecialImmTest.java

```diff
@@ -36,7 +36,7 @@
  * @library /test/lib
  * @requires os.arch == "aarch64"
  * @modules jdk.incubator.vector
- * @run testng/othervm -XX:UseSVE=0 -XX:-TieredCompilation -XX:CompileThreshold=100 compiler.vectorapi.VectorReplicateLongSpecialImmTest
+ * @run testng/othervm -XX:UseSVE=0 -XX:-TieredCompilation -XX:CompileThreshold=100 -XX:+IgnoreUnrecognizedVMOptions -XX:CompileCommand=MemLimit,*.*,0 compiler.vectorapi.VectorReplicateLongSpecialImmTest
```

Comment:
Does that mean we hit the bailout here and re-compilations hits the memory limit?

Reply:
Yes, exactly. We hit the memory limit later on, in register allocation. I suggest investigating in a separate issue, if there are no objections.

Reply:
Sounds good. Please file a separate issue then.